badges/baseBadge.js

```diff
@@ -25,6 +25,7 @@ module.exports = function dualBadge(text, options) {
         w2: box2.width,
         text1: box1.textPath,
         text2: box2.textPath,
-        color: options.color
+        color: options.color,
+        radius: options.radius,
```

Comment:
Sorry, one more comment. I just realized that you didn't add radius to the default options, so if the user doesn't pass it, this will be null, and cause issues. 

Could you please add a default radius option [here](https://github.com/lmarkus/openBadge/blob/7a6b114ece258c62db8fe0176fd005e96ca463c0/index.js#L27), and set it to '3' to maintain current behavior?

https://github.com/lmarkus/openBadge/blob/7a6b114ece258c62db8fe0176fd005e96ca463c0/index.js#L27